sdk/purview/purview-administration-rest/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "@azure/azure-sdk/ts-package-json-engine-is-present": "warn"
```

Comment:
Yay, most of the exceptions are now gone :)

For this last one, you could just use the override option: https://github.com/Azure/azure-sdk-for-js/blob/main/common/tools/eslint-plugin-azure-sdk/docs/rules/ts-package-json-engine-is-present.md